sdl_android/src/main/java/com/smartdevicelink/security/SdlSecurityBase.java

```diff
@@ -44,7 +44,8 @@ public void resetParams() {
     
     public void handleInitResult(boolean val) {
     	setInitSuccess(val);
-    	session.onSecurityInitialized();
+		if (session != null)
+			session.onSecurityInitialized();

```

Comment:
Right, but what I'm wondering is, can the security lib be initialized before a session is set? (session would be null, and the callback wouldn't happen at the changed code) If so, there would be no callback telling the session that security is initialized because it missed the only callback. If the security library is being reset, but the security library is still attempting to start, we should cancel the initialization process of the security library.